src/RemoteTech/RTLog.cs

```diff
@@ -5,46 +5,119 @@ namespace RemoteTech
 {
     public static class RTLog
     {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <value></value>
         private static readonly bool verboseLogging;
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <value></value>
```

Comment:
if we really just want to make these warnings go away, we can turn off xml docs rather than putting in a bunch of worthless code.